src/Sulu/Bundle/MarkupBundle/Markup/Link/LinkConfiguration.php

```diff
@@ -57,14 +57,29 @@ class LinkConfiguration
     #[Groups(['frontend'])]
     private $icon;
 
+    /**
+     * @var array<string, string>
+     */
+    #[Groups(['frontend'])]
+    private $targets = [
+        '_blank' => 'sulu_admin.link_blank',
+        '_self' => 'sulu_admin.link_self',
+        '_parent' => 'sulu_admin.link_parent',
+        '_top' => 'sulu_admin.link_top',
+    ];
```

Comment:
This kind of feels wrong for me using translation keys inside this class.

```suggestion
    /**
     * @var array<string, string>
     */
    #[Groups(['frontend'])]
    private $targets = [
        '_blank',
        '_self',
        '_parent',
        '_top',
    ];
```